cluster-scope/overlays/prod/emea/balrog/configmaps/service-catalog-k8s-plugin.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-catalog-k8s-plugin
+data:
+  env: emea
+  cluster: balrog
```

Comment:
Maybe a nit, however can we capitalize these? `$ENV` and `$CLUSTER` is easier to recognize in the script above than `$env` and `$CLUSTER`. Since capitals are convention for constants and env variables, it's easier to pick up these values come from a mounted configmap...